submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/refresh/SubmissionRefreshApiTest.kt

```diff
@@ -150,9 +155,11 @@ internal class SubmissionApiTest(private val tempFolder: TemporaryFolder) : Base
             submission.releaseTime = newReleaseDate
             submission.title = NEW_SUBTITLE
             submission.attributes = sortedSetOf(DbSubmissionAttribute(ATTR_NAME, NEW_ATTR_VALUE, 1))
-            submissionRepository.save(submission)
+            submissionDataRepository.save(submission)
         }
 
-        private fun getSubmissionDb(): DbSubmission = submissionRepository.readByAccNoAndVersionGreaterThan(ACC_NO, 0)!!
+        private fun getExtSubmission(): ExtSubmission = submissionRepository.getExtByAccNo(ACC_NO)
+
+        private fun getSubmissionDb(): DbSubmission = submissionDataRepository.getBasicWithAttributes(ACC_NO)
```

Comment:
we should not use DB submission at integration level

Reply:
This case is special because its purpose is testing the refresh feature which was actually created to reflect changes in the database. I understand your point but I think here we should make an exception.

Reply:
yes make sense